src/logger/logger.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  use(req: Request, res: Response, next: NextFunction) {
+    const ip = req.ip || req.connection.remoteAddress;
+    console.log(`I know your IP: ${ip}  baby`);
+    next();
+  }
+}
```

Comment:
You cand move this file and rename, other's related to "logger..." are useless in our case, because we create not resource, but middleware (simple function, not a logical component even)